app/src/main/res/drawable/ic_baseline_delete_24.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
```

Comment:
The dimension values should be in the dimens.xml file, same for the ones in ic_baseline_edit_24.xml and event_category_layout.xml

Reply:
Done